app/src/main/java/org/gdg/frisbee/android/adapter/EventAdapter.java

```diff
@@ -59,11 +60,24 @@
     private Context mContext;
     private LayoutInflater mInflater;
     private ArrayList<Item> mEvents;
+    private int mDeafultIcon;
 
     public EventAdapter(Context ctx) {
+        this(ctx, R.drawable.icon);
+    }
+    
+    public EventAdapter(Context ctx, @DrawableRes int defaultIcon) {
         mContext = ctx;
         mInflater = LayoutInflater.from(mContext);
         mEvents = new ArrayList<>();
+        mDeafultIcon = defaultIcon;
+    }
+
+    /**
+     * The default icon is used if there is no icon for an event.
+     */
+    public void setDeafultIcon(@DrawableRes int mDeafultIcon) {
```

Comment:
do we need a setter? better make it final and developers need to create a new adapter when showing a new tag